terminus/builders/osm_city_builder.py

```diff
@@ -0,0 +1,86 @@
+from geometry.point import Point
+
+from imposm.parser import OSMParser
+import math
+
+from models.city import City
+from models.street import Street
+from models.trunk import Trunk
+from models.block import Block
+from models.building import Building
+from models.ground_plane import GroundPlane
+from models.road import *
+
+import xml.etree.ElementTree as ET
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# For now exclude smaller roads, in the future we should draw each
+# type of street with different width/texture
+road_types = ['motorway', 'trunk', 'primary', 'secondary', 'tertiary',
```

Comment:
Not much of a fan of having constants like this, esp. for testing. I know is a less pythonic way, but can we move this to a method in `OsmCityBuilder`?